libs/components/src/next/form/form-control/FormControl.ts

```diff
@@ -77,6 +89,8 @@ export abstract class FormControl extends FormControlMixin(BaseLitElement) {
     abstract get validationTarget(): HTMLElement | undefined | null;
 
     resetFormControl(): void {
+        this.value = this.initialValue;
+        this.error = false;
```

Comment:
Deze zou ik niet resetten aangezien dit een value is die van buitenaf gezet kan worden.

Reply:
Besproken met Karim: voorlopig toch laten staan